package.json

```diff
@@ -31,8 +31,11 @@
     "test": "jest --color --ignoreProjects \"Gulp tasks\" --maxWorkers=50%",
     "test:screenshots": "node ./tasks/screenshot-components.mjs",
     "lint": "npm run lint:js && npm run lint:scss",
-    "lint:js": "eslint --cache --cache-location .cache/eslint --cache-strategy content --color --ignore-path .gitignore --max-warnings 0 \"**/*.{cjs,js,mjs}\"",
-    "lint:scss": "stylelint --cache --cache-location .cache/stylelint --cache-strategy content --color --ignore-path .gitignore --max-warnings 0 \"{app,src}/**/*.scss\""
+    "lint:js": "npm run lint:js:cli -- \"**/*.{cjs,js,mjs}\"",
+    "lint:js:cli": "eslint --cache --cache-location .cache/eslint --cache-strategy content --color --ignore-path .gitignore --max-warnings 0",
+    "lint:scss": "npm run lint:scss:cli -- \"{app,src}/**/*.scss\"",
+    "lint:scss:cli": "stylelint --cache --cache-location .cache/stylelint --cache-strategy content --color --ignore-path .gitignore --max-warnings 0",
+    "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\""
```

Comment:
This automatically installs the Git hooks [when you `npm install`](https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-scripts). This is [the lifecycle script recommended by Husky's documentation](https://typicode.github.io/husky/#/?id=install) (and we already have `postinstall` anyway).

We cannot run the recommended `husky install`, though. `husky` is a `devDependency`, as it's not needed to run the review app. Because of that, it won't be installed on Heroku, (or when running `npm install --omit=dev`) which means that `prepare` would fail if we were to run `husky install`. Instead [we can rely on the documented exception from Node](https://nodejs.org/api/modules.html#file-modules) to ensure a smooth installation, as proposed by [Husky's docs](https://typicode.github.io/husky/#/?id=with-a-custom-script).